src/core/MRApp.js

```diff
@@ -33,6 +33,8 @@ window.mobileCheck = function () {
     return mrjsUtils.display.mobileCheckFunction();
 };
 
+const GLOBAL_UPDATE_EVENTS = ['enterxr', 'exitxr', 'load', 'anchored', 'panelupdate', 'engine-started', 'resize'];
```

Comment:
add documentation note for why here and why order

Reply:
Ah when I said order I meant of the Systems, not the events.

Reply:
oh lol - all good